step/src/main/java/com/google/sps/servlets/LogInServlet.java

```diff
@@ -77,11 +77,11 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   }
 
   // Returns if the user logs in for the first time. If yes, return true. Otherwise, false.
-  private boolean getUserEmail(String email) {
+  private boolean getUserID(String id) {
```

Comment:
Can this method be named something more intuitive? getUserID sounds like I would expect an int or a long to be returned. I think something like getUserFirstLogin is a much more fitting name.